### PR TITLE
Allow activerecord <= 5.1

### PR DESCRIPTION
**Pull request by willsoto**
Now that rails 5.1 is out, this allows people to upgrade to it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,8 +31,10 @@ SchemaPlus::Enums is tested on:
 <!-- These lines are auto-generated by schema_dev based on schema_dev.yml -->
 * ruby **2.2.5** with activerecord **4.2**, using **postgresql**
 * ruby **2.2.5** with activerecord **5.0**, using **postgresql**
+* ruby **2.2.5** with activerecord **5.1**, using **postgresql**
```
Comment (lowjoel) [inline README.md]: @willsoto we should upgrade the Ruby versions to the latest stable in each branch, following what Rails tests against.

**Comment by willsoto** [comment]
@lowjoel Done. I updated them based on what I found [here.](https://github.com/rails/rails/blob/5c7e59a2bc7575273fb8067b7efce4af0f74affd/.travis.yml#L54-L58)

**Comment by willsoto** [comment]
@lowjoel ping for re-review please

**Comment by lowjoel** [comment]
Looks good. I'll merge, but I think only @ronen can release.

**Comment by willsoto** [comment]
@lowjoel Thank you for reviewing and merging. Would love to see this released soon since 5.1 is out 👍

**Comment by ronen** [comment]
Released.   Thanks both!

**Pull request merged**